ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -60,6 +63,13 @@ const CREDIT_LIMIT_DEFAULT_MIN = 300;
 const CREDIT_LIMIT_DEFAULT_MAX = 800;
 const CREDIT_LIMIT_DEFAULT_STEP = 50;
 
+const getUserStatus = (profile: Profile) => {
+  const {accessTierShortNames} = profile;
```

Comment:
nit: this could also be destructured in the function signature to save a line here.